dbt/adapters/athena/impl.py

```diff
@@ -825,16 +828,34 @@ def persist_docs_to_glue(
             glue_table_comment = table["Parameters"].get("comment", "")
             # Update description if it's different
             if clean_table_description != glue_table_description or clean_table_description != glue_table_comment:
-                updated_table["Description"] = clean_table_description
-                updated_table_parameters: Dict[str, str] = dict(updated_table["Parameters"])
-                updated_table_parameters["comment"] = clean_table_description
-                updated_table["Parameters"] = updated_table_parameters
-                need_udpate_table = True
+                table_input["Description"] = clean_table_description
+                table_parameters["comment"] = clean_table_description
+                need_to_update_table = True
+
+            # Get dbt model meta if available
+            meta: dict[str, Any] = model.get("config", {}).get("meta", {})
```

Comment:
To preserve consistency, can you put the type as `Dict[str, Any]` since we still support python 3.8 ?